### ISSUE TITLE
New Ideas/Features

### ISSUE THREAD

**Issue by ueg1990** [architv/soccer-cli#45]
I am still assuming this is not completely bug free and all and that there can still be improvements. But we also discuss any new ideas and features that we can add to the soccer-cli. :)

**Comment by architv** [comment]
Yes, surely. I have been thinking of some ideas to implement but haven't been getting enough time to do justice to them.

* **Team information(players)**: This should be fairly easy. The football data api [supports an endpoint for this](http://api.football-data.org/documentation).

* **Team Lineup for live matches**: The current API doesn't support this. We'll have to create our own resource for this.

* **A timeline of sorts for live matches**: Something like this ![](http://i.imgur.com/rqxo1JJ.png) would be very cool though I am not sure how this can be done.

**Comment by turrtle13** [comment]
An option for showing the time according to respective timezones would be better as UTC does not help while checking out the time of the live matches.

**Comment by carlosvargas** [comment]
@turrtle13 I made a PR #65 about this yesterday.

**Comment by akashwar** [comment]
I have been thinking of using this CLI projecting into configurable notifier tool that pushes desktop notification at a configurable time interval or over a goal scored on Linux or Windows user machine.
